norfair/tracker.py

```diff
@@ -1,5 +1,6 @@
 import math
 from typing import Callable, List, Optional, Sequence
+from copy import deepcopy
```

Comment:
These changes are unrelated and should not be on this PR. Please rebase interactive to remove them :)